src/components/notification/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import './notification.scss';
+
+const Notification = (props) => {
```

Comment:
@chingsley what does this do?

Reply:
I created a dismissable alert. The one the team is using doesn't have a dismissable implemented one.